packer/install-dependencies.sh

```diff
@@ -125,6 +125,11 @@ for AZURE_CNI_NETWORKMONITOR_VERSION in ${AZURE_CNI_NETWORKMONITOR_VERSIONS}; do
     pullContainerImage "docker" "containernetworking/networkmonitor:v${AZURE_CNI_NETWORKMONITOR_VERSION}"
 done
 
+AZURE_NPM_VERSIONS="1.0.18-rc2"
```

Comment:
these lines slightly changed in https://github.com/Azure/aks-engine/commit/5d1523de26ef19f719d00aeb6f98406f9afb5c5c#diff-62ed5765f6aaf1d428746b3d7b597100L125

Can you please rebase and add the new echo statement so this is included in release notes?

Reply:
done!